FirebaseScripts/CredentialsHelper.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from dotenv import load_dotenv
+
+dotenv_path = "./credentials.env"
+load_dotenv(dotenv_path)
+
+firebaseConfig = {
+    "apiKey": "",
+    "authDomain": "",
+    "databaseURL": "",
+    "projectId": "",
+    "storageBucket": "",
+    "messagingSenderId": "",
+    "appId": "",
+    "measurementId": ""
+}
+
+
+def get_fireBase_credentials():
+    try:
+
+        # Accessing variables.
+
+        apiKey = os.getenv('apiKey')
+        authDomain = os.getenv('authDomain')
+        databaseURL = os.getenv('databaseURL')
+        projectId = os.getenv('projectId')
+        storageBucket = os.getenv('storageBucket')
+        messagingSenderId = os.getenv('messagingSenderId')
+        appId = os.getenv('appId')
+        measurementId = os.getenv('measurementId')
+
+        if apiKey is None or authDomain is None or databaseURL is None or projectId is None or storageBucket is None or messagingSenderId is None or appId is None or measurementId is None:
```

Comment:
```suggestion
        credentials = [apiKey, authDomain, databaseURL, ...]
    
        if any(not(credential) for credential in credentials)):
```
Instead of checking each credential manually, create a list and use this shorthand.

Reply:
hey, @antrikshmisri resolved this issue please if anything else has to be done.